molgenis-metadata-manager/src/main/frontend/src/components/MetadataManagerAttributeEditForm.vue

```diff
@@ -0,0 +1,456 @@
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="col-6 col-form-label">Visible</label>
+            <div class="col checkbox-column">
+              <input v-model="visible" class="form-control" type="checkbox">
+            </div>
+          </div>
+        </div>
+
+        <div class="col-md-3">
+          <div class="form-group row">
+            <label class="col-6 col-form-label">Unique</label>
+            <div class="col checkbox-column">
+              <input v-model="unique" class="form-control" type="checkbox">
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="col-6 col-form-label">Read-only</label>
+            <div class="col checkbox-column">
+              <input v-model="readonly" class="form-control" type="checkbox">
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="col-6 col-form-label">Aggregatable</label>
+            <div class="col checkbox-column">
+              <input v-model="aggregatable" class="form-control" type="checkbox">
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col">
+          <div class="form-group">
+            <label>Computed value expression</label>
+            <textarea v-model="expression" class="form-control" rows="3"></textarea>
+          </div>
+        </div>
+        <div class="col">
+          <div class="form-group">
+            <label>Visible expression</label>
+            <textarea v-model="visibleExpression" class="form-control" rows="3"></textarea>
+          </div>
+        </div>
+        <div class="col">
+          <div class="form-group">
+            <label>Validation expression</label>
+            <textarea v-model="validationExpression" class="form-control" rows="3"></textarea>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style>
+  /*screen-md border on inner column when columns aligned next to each other*/
+  @media (min-width: 768px) {
+    .col-md-3.attribute-tree {
+      border-right: solid black thin;
+    }
+  }
+</style>
+
+<script>
+  import AttributeTree from './generic-components/AttributeTree'
+  import { mapState, mapGetters } from 'vuex'
+  import {
+    SET_SELECTED_ATTRIBUTE_ID,
+    UPDATE_EDITOR_ENTITY_TYPE_ATTRIBUTE,
+    DELETE_SELECTED_ATTRIBUTE,
+    UPDATE_EDITOR_ENTITY_TYPE_ATTRIBUTE_ORDER
+  } from '../store/mutations'
+
+  import { CREATE_ATTRIBUTE } from '../store/actions'
+  import { getConfirmBeforeDeletingProperties } from '../store/getters'
+
+  import Multiselect from 'vue-multiselect'
+
+  export default {
+    name: 'metadata-manager-attribute-edit-form',
+    methods: {
+      deleteAttribute (selectedAttribute) {
+        this.$swal(getConfirmBeforeDeletingProperties(selectedAttribute.label)).then(() => {
```

Comment:
in main.js:
```js
import VueSweetAlert from 'vue-sweetalert'
Vue.use(VueSweetAlert)
```

This kind if inclusion exposes sweet alert in all child components.
Similar to vuex, where you do `this.$store...`